recipe/meta.yaml

```diff
@@ -23,6 +23,7 @@ requirements:
     - setuptools-scm >=6.2
     - pip
   run:
+    - build >=1.0.3
```

Comment:
This should not be a runtime dependency. Probably something to report upstream.

Reply:
Got introduced in https://github.com/chroma-core/chroma/pull/1574. Probably they wanted to add it to `requirements.txt` only.

Reply:
Made an upstream PR https://github.com/teleprint-me/chroma/pull/11 and will add it to the feedstock.